### DISCUSSION TITLE
Grab 5 records

### DISCUSSION THREAD

**Discussion by dimdam** [Power-Components/livewire-powergrid#587] [Q&A]
How can i view only 5 records from the database and sort them based on specific column?

**Comment by akulmehta** [comment]
See the use for `$perPage` in the docs in this section https://livewire-powergrid.com/#/table/features-setup?id=showperpage 

For sorting you can customize like https://livewire-powergrid.com/#/table/component-settings?id=sort-by-field-and-direction
